application/F3DConfigFileTools.cxx

```diff
@@ -82,6 +68,21 @@ std::vector<fs::path> GetConfigPaths(const std::string& configSearch)
 
   return paths;
 }
+
+//----------------------------------------------------------------------------
+void F3DConfigFileTools::PrintConfigInfo(const std::vector<fs::path>& configPaths)
+{
+  for (const fs::path& path : F3DConfigFileTools::GetConfigPaths("config"))
+  {
+    if (fs::exists(path))
+    {
+      f3d::log::info("Config file found: ", path);
+    }
+    else
+    {
+      f3d::log::info("Candidate config file not found: ", path);
+    }
+  }
```

Comment:
I'd say we can get rid of this method and log specifically in `ReadConfigFiles` and the `print_config_info` command

Reply:
I dont think that works ?

Reply:
ha right, sure, that works too :)